src/config/dynamic-theme-fixes.config

```diff
@@ -17355,10 +17355,11 @@ body > .oo-ui-windowManager .vega .marks
 .minerva-footer-logo img
 .music-symbol
 .tool.tool-button[src$="background-image:"]
-.wikiEditor-ui .oo-ui-iconElement-icon
+.oo-ui-iconElement-icon
```

Comment:
Does this cover the general `.oo-ui-icon-close` element?

Reply:
The only case I've seen so far is the close video button in MinervaNeue skin and it has `oo-ui-iconElement-icon` class.

Reply:
Lgtm then